src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div
+    data-qa="big"
+    class="search big-bar">
+      <img
+      class="image-big"
+      src="images/Search.svg"
+      alt="search">
+      <input
+      class="search-bar"
+      data-qa="keypress"
+      type="text"
+      name="big-search"
+      placeholder="Try 'Los Angeles'">
```

Comment:
```suggestion
      <input
        class="search-bar"
        data-qa="keypress"
        type="text"
        name="big-search"
        placeholder="Try 'Los Angeles'"
      >
```
fix formatting everywhere, move closing brace to the new line